tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_container_validation.py

```diff
@@ -875,6 +875,27 @@ def test_valid_containers(
             auto_type_section=AutoSection.NONE,
             validity_error=[EOFException.MISSING_TYPE_HEADER, EOFException.UNEXPECTED_HEADER_KIND],
         ),
+        Container(
+            name="no_type_section_2_codes",
+            sections=[
+                Section.Code(Op.INVALID),
+                Section.Code(Op.INVALID),
+            ],
+            auto_type_section=AutoSection.NONE,
+            auto_data_section=False,
+            expected_bytecode="ef0001 020002 0001 0001 00 fefe",
+            validity_error=[EOFException.MISSING_TYPE_HEADER, EOFException.UNEXPECTED_HEADER_KIND],
+        ),
+        Container(
+            name="no_type_section_data_section",
+            sections=[
+                Section.Code(Op.INVALID),
+                Section.Data("0xda"),
+            ],
+            auto_type_section=AutoSection.NONE,
+            expected_bytecode="ef0001 020001 0001 040001 00 feda",
+            validity_error=[EOFException.MISSING_TYPE_HEADER, EOFException.UNEXPECTED_HEADER_KIND],
+        ),
         Container(
```

Comment:
```suggestion
        Container(
            name="no_type_section_container_section",
            sections=[
                Section.Code(Op.INVALID),
                Section.Container(
                    Container(
                        sections=[
                            Section.Code(code=Op.RETURNCONTRACT[0](0, 0)),
                            Section.Container(container=Container.Code(code=Op.STOP)),
                        ],
                    )
                ),
            ],
            auto_type_section=AutoSection.NONE,
            expected_bytecode="ef0001 020001 0001 030001 0032 040000 00 fe"
            "ef0001 010004 020001 0006 030001 0014 040000 00 00800002 60006000ee00"
            "ef0001 010004 020001 0001 040000 00 0080000000",
            validity_error=[EOFException.MISSING_TYPE_HEADER, EOFException.UNEXPECTED_HEADER_KIND],
        ),
        Container(
```

Just for the sake of completeness.